simulation.cpp

```diff
@@ -386,8 +395,8 @@ void test_simulation() noexcept//!OCLINT test may be many
         change_nth_ind_net(s, worst_ind, worst_net);
         auto resp_worst = response(get_nth_ind(s, worst_ind))[0];
 
-        assert(resp_best == get_current_env_value(s));
-        assert(resp_worst != get_current_env_value(s));
+        assert(resp_best == optima_is_response_of_grn_first_ind(s.get_pop()));
```

Comment:
The environmental function are set up to work with inputs, I don't think feeding them a population is going to work.

Reply:
Well spotted, 
That is exactly what I thought was the problem when you asked me to change something. I will try to think of a workaround.